.github/workflows/add-lang.yml

```diff
@@ -17,12 +17,14 @@ jobs:
           ISSUE_TITLE: ${{ github.event.issue.title }}
           ISSUE_NUM: ${{ github.event.issue.number }}
         run: |
-          NEW_LANG=$(sed 's/.*\[\([^]]*\)].*/\1/' <<< "$ISSUE_TITLE")
-          if [[ $NEW_LANG =~ ^[a-z]{2}(_[A-Z]{2})?$ ]]; then
-            echo "new_lang=$NEW_LANG" >> $GITHUB_ENV
+          LANG_CODE=$(sed 's/.*\[\([^]]*\)].*/\1/' <<< "$ISSUE_TITLE")
+          FULL_LANG=$(sed 's/.*<\([^>]*\)>.*/\1/' <<< "$ISSUE_TITLE")
+          if [[ $LANG_CODE =~ ^[a-z]{2}(-[A-Z]{2})?$ || $FULL_LANG =~ <[^>]+> ]]; then
```

Comment:
Note to self: the `$FULL_LANG` check is wrong (`$FULL_LANG` itself doesn't have < and > characters).